src/inc/til/string.h

```diff
@@ -169,7 +169,7 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
     // and 5x or more for long strings (128 characters or more).
     // See: https://github.com/microsoft/STL/issues/2289
     template<typename T, typename Traits>
-    bool equals(const std::basic_string_view<T, Traits>& str1, const std::basic_string_view<T, Traits>& str2) noexcept
+    bool equals(const std::basic_string_view<T, Traits>& lhs, const std::basic_string_view<T, Traits>& rhs) noexcept

```

Comment:
As mentioned before, with `/permissive-` and true two phase lookup, templates are actually parsed! This means such programming errors can't pass accidentally anymore, because the compiler will actually complain that the code is invalid even if the template isn't used anywhere.

Reply:
...wow this template just was never compiled lol